docs/src/QuickStart/running-qat.md

```diff
@@ -0,0 +1,414 @@
+# Running QAT
+
+In the [next section](../UsingQAT/TargetingQIR.md) we will discuss the
+individual validators and adaptors. However, for the sake of getting started
+quickly, in this section, we will take the reader through the basic usage of
+QAT, its configuration and how to use selected adaptors using examples.
+
+## Basic usage
+
+The easiest way to use QAT is by running it with a target definition:
+
+```sh
+qat --target-def path/to/target-def.yaml -S -o output.ll input.ll
+```
+
+The file `target-def.yaml` sets the behaviour of QAT, whereas `-S` tells QAT to
+emit human readable LLVM code (as opposed to the binary format). The output file
+is specified by `-o` and finally, a list of one or more input files is given for
+QAT to process. Providing more than a single input file, QAT will link these
+into a single intermediate representation.
+
+To get a full list of the commandline arguments and flags, run `qat -h` or
+`qat --help`.
+
+## Target configuration
```

Comment:
You talk about target definition above but name it target configuration here. Calling it target configuration in the section above will make it clearer. It also seems like this should be a subsection of Basic Usage.